### ISSUE TITLE
lost cookie in ProxyServlet

### ISSUE THREAD

**Issue by snowlidong** [mitre/HTTP-Proxy-Servlet#110]
i use ProxyServlet(1.8) to proxy my request.i can not get cookie in real server using com.sun.jersey.api.client.Client to send request. the cookie value looks like: 
"!Proxy!JSESSIONID=qvgukk6gb9c156kaqytesi67;Version=1"
i debug in source, found that function:
 `getRealCookie(String cookieValue)`  cannot get cookie.

**Comment by huixiusnow** [comment]
I got the same problem. There is a bug in row502.` String cookies[] = cookieValue.split("; ");` 

When it splits the param "cookieValue"--`"!Proxy!JSESSIONID=qvgukk6gb9c156kaqytesi67;Version=1"`, we get one part , the result still be `"!Proxy!JSESSIONID=qvgukk6gb9c156kaqytesi67;Version=1"`. Next ,in "for" sentence ,it will use` "="` to split the result. Then we get three parts, so we cannot get the real cookie we want.

I looked over the `com.sun.jersey.core.header.reader.CookiesParser`, the function "parseCookies(String header)", it uses "`header.split("[;,]");`" to split the cookie. 

I modified the ProxyServlet like"`cookieValue.split("[;,]");`", then it works!

Hope you fix this bug soon!
